src/Base64.tsx

```diff
@@ -44,6 +44,19 @@ function Base64(): JSX.Element {
         setSeparatedLines(newSeparatedLines)
     }
 
+    function getBase64Text(): string {
+        return base64Encode(textInputValue, encodingName)
+    }
+
+    function copyBase64ToClipboard() {
+        navigator
+            .clipboard
+            .writeText(getBase64Text())
+            // CANDO Add some kind of notification
+            // (could maybe also be used for other stuff, like info text when a click on "encode lines separately" automcatically unchecks "wrap" option)
```

Comment:
I like that idea a lot. Maybe remove the cando and create a 'ticket' for that

Reply:
Created #37